src/desktop/analytics/account_creation.js

```diff
@@ -55,76 +55,63 @@ $(document).on("click", ".auth-signup-facebook", e => {
   })
 })
 
-// 3. After landing back on Artsy send the tracking call and expire the cookie
-if (Cookies.get("analytics-signup")) {
-  var data = JSON.parse(Cookies.get("analytics-signup"))
-  Cookies.expire("analytics-signup")
-
-  if (sd.CURRENT_USER) {
-    trackAccountCreation({
-      ...data,
-      user_id: sd.CURRENT_USER.id,
-    })
-  }
+analyticsHooks.on("auth:login", (options = {}) => {
+  analytics.track("Successfully logged in", options)
```

Comment:
Leaving the legacy event for the moment - not sure if this is actually used, we can double check in segment and potentially delete.

Reply:
These events were not previously firing because of a typo that moved them all inside a `Cookies.get("analytics-signup"` conditional.